test/integration/functional_test.go

```diff
@@ -316,6 +337,26 @@ func validateImageCommands(ctx context.Context, t *testing.T, profile string) {
 		checkImageExists(ctx, t, profile, taggedImage)
 	})
 
+	// docs: Try to load image already loaded and make sure `minikube image load --daemon` works
+	t.Run("ImageReloadDaemon", func(t *testing.T) {
```

Comment:
The test before this does an identical daemon image load, so this test only needs to do a single image load for it to be a reload.